README.md

```diff
@@ -1,19 +1,238 @@
+merged into the RFC repository as a markdown file. At that point the RFC is
+"active" and may be implemented with the goal of eventual inclusion into OBS.'
+It is recommended that no work start on implementation until an RFC is accepted.
+
+  - Fork the RFC repo [RFC repository]
+  - Copy `0000-template.md` to `text/0000-my-feature.md`. Make sure the title
+    is descriptive. RFC number should match the next PR number.
+  - Fill in the RFC. Put care into the details: RFCs that do not present
+    convincing motivation, demonstrate lack of understanding of the design's
+    impact, or are disingenuous about the drawbacks or alternatives tend to
+    be poorly-received.
+  - Submit a pull request to the [RFC repository]. As a pull request the RFC
+    will receive design feedback from the larger community, and the author
+	should be prepared to revise it in response.
+  - Build consensus and integrate feedback. RFCs that have broad support are
+    much more likely to make progress than those that don't receive any
+    comments. Feel free to reach out to the project developers to get
+    help identifying issues and obstacles.
+  - The project team will discuss the RFC pull request, as much as possible in the
+    comment thread of the pull request itself. Offline discussion will be
+    summarized on the pull request comment thread.
+  - RFCs rarely go through this process unchanged, especially as alternatives
+    and drawbacks are shown. You can make edits, big and small, to the RFC to
+    clarify or change the design, but make changes as new commits to the pull
+    request, and leave a comment on the pull request explaining your changes.
+    Specifically, do not squash or rebase commits after they are visible on the
+    pull request.
+  - At some point, a member of the team will propose a "motion for final
+    comment period" (FCP), along with a *disposition* for the RFC (merge, close,
+    or postpone).
+    - This step is taken when enough of the tradeoffs have been discussed that
+    the team is in a position to make a decision. That does not require
+    consensus amongst all participants in the RFC thread (which is usually
+    impossible). However, the argument supporting the disposition on the RFC
+    needs to have already been clearly articulated, and there should not be a
+    strong consensus *against* that position outside of the team. Team
+    members use their best judgment in taking this step, and the FCP itself
+    ensures there is ample time and notification for stakeholders to push back
+    if it is made prematurely.
+    - For RFCs with lengthy discussion, the motion to FCP is usually preceded by
+      a *summary comment* trying to lay out the current state of the discussion
+      and major tradeoffs/points of disagreement.
+  - The FCP lasts ten calendar days, so that it is open for at least 5 business
+    days. This way all stakeholders have a chance to lodge any final objections
```

Comment:
i realize "stakeholders" is being used here colloquially, but I wonder if we should use a different word here anyway, such as "interested parties".

Reply:
I like this, I'll make the changes tonight. I left it in where it was being used loosely fro lack of a better term, but I think interested parties is more descriptive for the audience we're actually targeting.